test/conformance/testing/uur/fixtures.h

```diff
@@ -335,6 +364,10 @@ struct urUSMDeviceAllocTestWithParam : urQueueTestWithParam<T> {
 
     void SetUp() override {
         UUR_RETURN_ON_FATAL_FAILURE(uur::urQueueTestWithParam<T>::SetUp());
+        if (!GetDeviceInfo<bool>(this->device,
+                                 UR_DEVICE_INFO_USM_DEVICE_SUPPORT)) {
+            GTEST_SKIP() << "Device USM in not supported";
+        }
```

Comment:
I suggest to make this similar to the comment above

Reply:
Done.